.github/workflows/generator_dart.yaml

```diff
@@ -14,15 +14,17 @@
 
 name: Client Generator - Dart
 permissions: read-all
+
 # Run on PRs and pushes to the default branch.
 on:
   push:
     branches: [main]
   pull_request:
     branches: [main]
+
 jobs:
   generator-build:
-    runs-on: ubuntu-24.04
+    runs-on: ubuntu-latest
```

Comment:
We prefer to keep these pinned.  When GitHub redefines "latest" to 26.04 we don't want all the builds to break and have to scramble. We would rather upgrade at a time of our choosing.